### DISCUSSION TITLE
wyze_OG

### DISCUSSION THREAD

**Discussion by ju5t3nc4s3** [HclX/WyzeHacks#155] [Ideas]
I started reverse engineering the  new OG model . its using a arm chip that is the same used in a open development board. So i have located the soc datasheet and many other good docks. the dev board has a SDk and is flashed over arduino.I have been able to get a shell throw uart , have located a way to do a firmware update over URL. and uart. also cmd to enable rtsp. im about to the end of my knowledge , not knowing much coding. (hardware hacker)  dont know whoes interested but need imput. this is a good cheep device at 20$ on amazon.

**Comment by ntmera** [comment]
Interested, particularly in cmd to enable RTSP 😆
